certified-connectors/Asite/apiDefinition.swagger.json

```diff
@@ -0,0 +1,799 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Asite",
+    "description": "Asite connector helps to built a connection between two systems for file exchange. Connector collaborate both systems by upload and download file files based on triggers. ",
+    "version": "1.0.0",
+	"contact": {
+      "name": "Asite Solutions Pvt. Ltd.",
+      "url": "https://www.asite.com/contact-support",
+      "email": "support@asite.com"
+    }
+  },
+  "host": "integrationqaak.asite.com",
+  "basePath": "/api",
+  "x-ms-connector-metadata": [
```

Comment:
@asitedeveloper Please fix indentation issues in all the files of your connector

Reply:
@sriyen-msft We have fixed indentation in all files of our connector. let us know if need to required any change.